nemo_skills/evaluation/metrics/map_metrics.py

```diff
@@ -30,10 +30,12 @@
 from nemo_skills.evaluation.metrics.math_metrics import MathMetrics
 from nemo_skills.evaluation.metrics.mrcr_metrics import MRCRMetrics
 from nemo_skills.evaluation.metrics.ruler_metrics import RulerMetrics
+from nemo_skills.evaluation.metrics.simpleqa_metrics import SimpleQAMetrics
 
 METRICS_MAP = {
     "math": MathMetrics,
     "hle": functools.partial(MathMetrics, compute_no_answer=False, answer_key="generation"),
+    "simpleqa": functools.partial(SimpleQAMetrics, compute_no_answer=False),
```

Comment:
if you're using a custom class anyway, let's just set compute_no_answer to False there by default and use this without functools.partial